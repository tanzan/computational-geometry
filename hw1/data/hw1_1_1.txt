-4 4 2 -2
4
0 0
5 -5
1 1
-1 -1