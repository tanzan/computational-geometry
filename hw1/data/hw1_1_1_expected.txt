ON_SEGMENT
ON_LINE
LEFT
RIGHT